### DISCUSSION TITLE
Feature Request: Add Current Date and Time as File Attributes in LosslessCut

### DISCUSSION THREAD

**Discussion by fabriconti** [mifi/lossless-cut#2207] [Ideas]
I would like to propose a feature enhancement for LosslessCut. 

It would be incredibly useful if you could implement an option that allows users to automatically write the current date and time as attributes for the files generated through the application. 

This feature would greatly aid in organizing and managing video files, especially for those of us who handle multiple projects simultaneously.

**Comment by mifi** [comment]
What do you mean by "file attributes"? can you provide a technical explanation of exactly what this means?

**Reply by fabriconti** [reply]
I refer to creation time, or modification time: The date and time when the exported file(s) was created or last modified. 
If I export now files from LC, creation time filesystem attribute will be same of original file. Best option (for me) if exported files take creation datetime (or modification time) from current timestamp.

This capability would allow for better tracking of when a exported file was processed or created, thereby improving workflow efficiency and file management practices. 

Keeping Explorer (or Total Commander, or Everything.exe) with the file view sorted by 'last modified date' in descending order, I can quickly find the files I need.

**Reply by mifi** [reply]
There's a setting called "Set file modification date/time of output files to: Current time" - doesn't that solve this problem?

**Reply by fabriconti** [reply]
Yes! Reading the Italian translation, it wasn't clear that this was the option I was looking for. Thank you!

![image](https://github.com/user-attachments/assets/9cdefe6b-17d2-48d2-878a-3fb1b600c172)
